docs/source/peers/peers.md

```diff
@@ -312,40 +360,67 @@ peer cannot, on its own, update the ledger --- updating requires the consent of
+3-phase process which, through the Fabric Gateway service in v2.4, ensures that all  
+peers in the blockchain network keep their ledgers consistent with each other:
+
+1. In the first phase, the client application sends a transaction proposal to a
+gateway peer, preferably in its own organization. Assuming the application has
+delegated transaction endorsement to the Fabric Gateway service, the gateway peer
+validates (runs) the transaction and forwards it to endorsing peers in the organizations
+required to satisfy the endorsement policy. These peers also validate the transaction
+and return the endorsement results to the gateway peer, without updating their ledgers.
+2. In the second phase, the gateway peer collects the endorsement results from
+the peers and packages them into blocks. If endorsed, the gateway service sends the
+transaction to the client application for signing, which signs and returns it
+to the gateway peer.
+3. In the third and final phase, the signed transaction (blocks) are distributed
+to the endorsing peers, which validate the transaction.
+
+In addition to the client application and gateway peers, orderer nodes play a
+critical role in transactions that update the ledger. Before committing transactions
+to the ledger, orderer nodes do just that --- they order all submitted transactions
+to ensure a consistent and accurate sequence of transactions on the shared ledger.
+Note that blockchain manages inconsistent ledgers through a process called forking,
+and the subsequent pruning (discarding and archiving) of the invalidated forks.
 
 ### Phase 1: Proposal
 
-Phase 1 of the transaction workflow involves an interaction between an
-application and a set of peers --- it does not involve orderers. Phase 1 is only
-concerned with an application asking different organizations' endorsing peers to
-agree to the results of the proposed chaincode invocation.
-
-To start phase 1, applications generate a transaction proposal which they send
-to each of the required set of peers for endorsement. Each of these *endorsing peers* then
-independently executes a chaincode using the transaction proposal to
-generate a transaction proposal response. It does not apply this update to the
-ledger, but rather simply signs it and returns it to the application. Once the
+Phase 1 of the transaction workflow involves an interaction between a client
+application and a set of peers --- orderers are not involved yet. In Phase 1,
+the client application initiates a request to endorsing peers, through the
+Fabric Gateway service, to return their responses to the proposed transaction.
+
+To begin phase 1, the client application generates a transaction proposal and
+sends it to a gateway peer in its own organization. The gateway peer with
+the highest block height is selected; if none are discovered, the gateway
+service moves on to select a gateway peer in another organization.
```

Comment:
It does not "move on to select a gateway peer in another organization".

Reply:
so what happens then if none are discovered. or is that not a viable scenario.

Reply:
The gateway peer has ability to execute/endorse on the local peer itself (assuming the chaincode is installed locally), and therefore the local peer is always implicitly "discovered".

Reply:
deleting "if none are discovered ..."